client/src/components/myBookings/myBookings.jsx

```diff
@@ -53,10 +54,12 @@ class MyBookings extends React.Component {
   render() {
     return (
       <div className='my-bookings'>
-        <MyBookingsHeader heading={'Active Bookings'} />
-        {
-          this.state.bookings.map((booking, idx) => <ParkingSpot key={idx} image={booking.photo_url} address={booking.address} date={booking.date} time={booking.time} total={booking.total} onIconClick={this.onClick} ActionIcon={Trash} />)
-        }
+        <PageHeader title='Active Bookings' linkto='/rent'/>
```

Comment:
not sure if you need the back button, but if the back button is not set up to be visible (isBackButtonVisible), 'linkto' wouldn't do anything so you don't have to assign it here.

Reply:
@jenniferjinyoungyang Yup. I saw your PR and I understand what's happening. I dont need the back button actually. But do you think it will be more consistent with other components if I leave it there?

Reply:
@jenniferjinyoungyang Never mind. I saw other components dont have a back btn. So I will remove the back button too.